akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -211,20 +211,41 @@ akka.cluster.auto-down-unreachable-after = 120s
 This means that the cluster leader member will change the `unreachable` node
 status to `down` automatically after the configured time of unreachability.
 
-This is a naïve approach to remove unreachable nodes from the cluster membership. It
-works great for crashes and short transient network partitions, but not for long network
-partitions. Both sides of the network partition will see the other side as unreachable
-and after a while remove it from its cluster membership. Since this happens on both
-sides the result is that two separate disconnected clusters have been created. This
-can also happen because of long GC pauses or system overload.
+This is a naïve approach to remove unreachable nodes from the cluster membership.
+It can be useful during development but in a production environment it causes
+a breakdown of the cluster. When a network partition occurs, both sides of the
```

Comment:
"breakdown" is not good wording, there are systems which would work totally with this, esp d-data based ones etc.

Rather explain that it may result in unexpected downing and unstable cluster memersbip

Reply:
My understanding (correct me if I am wrong) is that with auto-downing, any time there is a partition (due to network issues, GC pauses etc), nodes will be removed from the cluster. When that happens, those nodes will not rejoin the cluster when the problem resolves. This means that over time the cluster will breakdown into smaller and smaller clusters, eventually resulting in a bunch of single node clusters. I am not sure that any system would work properly with this configuration.

Reply:
It's correct, it will be a breakdown into smaller and smaller pieces until there are only single node clusters remaining. Not very useful for any real system. Possibly one could manually or automatically cleanup and shutdown/restart these smaller pieces but let's not recommend that.

Reply:
In situations where you're running in something that keeps apps/pods/vms running and a Down means a jvm shutdown (one would have to impl this though), then the external restarting could end up healing such cluster. It's not without "external help", but people do run in k8s or similar things which do this.

If you're fine with the rewording proposed I'm ok with it, but there are cases where it could be "good enough", though addmitably, it's pretty rare cases I guess.

Reply:
That is a misunderstanding. Auto-down will only down other nodes, not itself. Since there is a network partition the Down signal will never reach the other side, which also keeps running.

Reply:
So what's the verdict? Reword, or leave as is?

Reply:
> That is a misunderstanding. Auto-down will only down other nodes, not itself. Since there is a network partition the Down signal will never reach the other side, which also keeps running.

Hm right, my bad.

Reply:
perhaps: "in a production environment it will eventually breakdown the cluster"

Reply:
Reworded as suggested.